modules/mass.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * This module adds MASS support to Prebid.js.
+ */
+
+import { config } from '../src/config.js';
+import { getHook } from '../src/hook.js';
+import find from 'core-js-pure/features/array/find.js';
+
+let listenerAdded = false;
+let massEnabled = false;
+let bootloaderUrl = 'https://cdn.massplatform.net/bootloader.js';
+
+const massBids = {};
+
+init();
+config.getConfig('mass', config => init(config.mass));
+
+/**
+ * Module init.
+ */
+function init(cfg = {}) {
+  if (cfg.enabled === false) {
+    if (massEnabled) {
+      massEnabled = false;
+      getHook('addBidResponse').getHooks({hook: addBidResponseHook}).remove();
+    }
+  } else {
+    if (!massEnabled) {
+      getHook('addBidResponse').before(addBidResponseHook);
+      massEnabled = true;
+    }
+  }
+
+  if (cfg.bootloaderUrl) {
+    bootloaderUrl = cfg.bootloaderUrl;
+  }
+}
+
+/**
+ * Before hook for 'addBidResponse'.
+ */
+export function addBidResponseHook(next, adUnitCode, bid) {
+  if (!isMassBid(bid)) {
+    return next(adUnitCode, bid);
+  }
+
+  const bidRequest = find(this.bidderRequest.bids, bidRequest =>
+    bidRequest.bidId === bid.requestId
+  );
+
+  massBids[bid.requestId] = {
+    bidRequest,
+    bid,
+    adm: bid.ad
+  };
+
+  bid.ad = '<script>window.top.postMessage({massBidId: "' + bid.requestId + '"}, "*");\x3c/script>';
+
+  addListenerOnce();
+
+  next(adUnitCode, bid);
+}
+
+/**
+ * Check if a bid is MASS.
+ */
+function isMassBid(bid) {
+  // the deal ID must start with MASS:
```

Comment:
should be `META_MASS` now, right?

```suggestion
  // the deal ID must start with META_MASS:
```